src/test/ide.test.ts

```diff
@@ -565,4 +565,43 @@ test('can jump from colon in scripts section to wireit config', async ({
   });
 });
 
+test('can handle a script with  from colon in scripts section to wireit config', async ({
```

Comment:
typo in this description I think?

Reply:
Wasn't meant to be part of this change, removed!